sci-libs/mc/mc-1.5.ebuild

```diff
@@ -26,59 +25,38 @@ RDEPEND="
 	sci-libs/umfpack
 	virtual/blas
 	virtual/lapack"
-DEPEND="
-	${RDEPEND}
+DEPEND="${RDEPEND}"
+BDEPEND="
 	doc? (
-		media-gfx/graphviz
 		app-doc/doxygen
-		)"
-
-S="${WORKDIR}"/${PN}
+		media-gfx/graphviz
+	)"
 
 PATCHES=(
 	"${FILESDIR}"/1.4-superlu.patch
 	"${FILESDIR}"/1.4-overflow.patch
 	"${FILESDIR}"/1.4-multilib.patch
 	"${FILESDIR}"/1.4-doc.patch
 	"${FILESDIR}"/${P}-unbundle.patch
-	)
+)
 
 src_prepare() {
+	default
 	sed \
 		-e 's:AMD_order:amd_order:g' \
 		-e 's:UMFPACK_numeric:umfpack_di_numeric:g' \
 		-e 's:buildg_:matvec_:g' \
 		-i configure.ac || die
-	autotools-utils_src_prepare
+	eautoreconf
 }
 
 src_configure() {
-	local fetk_include
-	local fetk_lib
-	local myeconfargs
-
-	use doc || myeconfargs+=( --with-doxygen= --with-dot= )
-
-	fetk_include="${EPREFIX}"/usr/include
```

Comment:
Huh, that's a huge chunk gone? Wasn't needed at all?

(Sure it's not falling back on anything bundled?)

Reply:
I did compare shared libraries used, and was still same amount. I did start to doubt when I moved on to punc given things were different there.

Reply:
It's also same size so I'd assume nothing got added, but I admit the configure messages worry me. Maybe worth more looking into.